fuzzy-pickles-derive/src/lib.rs

```diff
@@ -67,8 +67,9 @@ fn impl_visit(ast: &syn::MacroInput) -> quote::Tokens {
             fn visit<V>(&self, v: &mut V)
                 where V: Visitor
             {
-                v.#method_name(self);
-                #visit_fields;
+                if let Control::Continue = v.#method_name(self) {
```

Comment:
Any particular reason you went with an `if let` instead of straight equality?

Reply:
Nope, just coding after a long day ^^